src/styles/block/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main;
+  }
+  &__box {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 0;
+    color: $secondary;
+  }
+  &-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+    &__text {
+      color: #060b35;
+      text-align: right;
+      font-size: 10px;
+      font-weight: 400;
+      margin: 0;
+    }
+  }
+  &-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    margin-bottom: 0;
+    &__title {
+      margin-top: 0;
+      font-size: 12px;
```

Comment:
`line-height: 18px` and `font-size: 12px;` are duplicated, so please move all duplicated styles to the variables